lib/sms/sms.c

```diff
@@ -41,6 +42,9 @@ LOG_MODULE_REGISTER(sms, CONFIG_SMS_LOG_LEVEL);
 /** @brief AT notification informing that SMS client has been unregistered. */
 #define AT_SMS_UNREGISTERED_NTF "+CMS ERROR: 524"
 
+#define MODEM_CFUN_NORMAL 1
```

Comment:
Seeing these CFUN values defined in so many places, I can't stop thinking that should we define these in `nrf_modem_lib.h` which defines `NRF_MODEM_LIB_ON_CFUN`. Even that doesn't have comments on what mode values actually are although they are obvious to many of us. `nrf_modem_lib_init_cb` has comments on the parameters but `nrf_modem_lib_at_cfun_cb`.
This is maybe something for a separate PR but raised my attention now that we need to add these defines in several places.

Reply:
Good point.

Reply:
I was thinking about that too, but let's take that in a separate PR.